src/Repository/SiteRepository.php

```diff
@@ -19,6 +20,18 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, Site::class);
     }
 
+    public function findByRootDirAndServer(string $rootDir, Server $server)
```

Comment:
Return type would be nice here?

Reply:
Added. But ´getResult()` retuerns `mixed` so not very usefull